control/control_loop.go

```diff
@@ -281,3 +282,13 @@ func (l *Loop) Stop() {
 func (l *Loop) GetConfig(ctx context.Context) Config {
 	return l.cfg
 }
+
+// GetTuning returns the current tuning value.
+func (l *Loop) GetTuning(ctx context.Context) bool {
+	return l.tuning
```

Comment:
is there a way we could have this logic in the PID block?

Reply:
like add it to the PID config? I can for sure

Reply:
since we will be discussing changes to the control config in a couple weeks, I am thinking I'll leave this for now